### PR TITLE
Update test lvmlib to use argparse

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
Signed-off-by: Mark Syms <mark.syms@citrix.com>

### CODE REVIEW

**Comment by MarkSymsCtx** [comment]
Just a modernisation of this test support library which was getting in the way of doing some fault tracing.

File: tests/lvmlib.py [hunk 1]
```diff
@@ -72,51 +72,53 @@ def fake_lvchange(self, args, stdin):
...
+        active = not args.inactive
+        assert args.zero in ['y', 'n']
+        zeroed = args.zero == 'y'
 
         self.get_volume_group(vgname).add_volume(
-            options.name,
-            int(options.size_mb),
-            options.tag,
+            args.name,
+            int(args.size_mb),
```
Comment (rosslagerwall) [inline tests/lvmlib.py:101]: Not that it matters particularly but it might be preferable to `parser.add_argument("-L", dest='size_mb', type=int)` instead.

**Review by rosslagerwall — DISMISSED** [review]

File: tests/lvmlib.py (hunk 1 above)
Comment (MarkSymsCtx) [reply tests/lvmlib.py:101]: Yes, sensible, it was a basic search replace for the most part but that would be a further improvement in maintainability here.

**Review by TimSmithCtx — APPROVED** [review]

**Review by edwintorok — DISMISSED** [review]
This is all test code so I assume any failures would've shown up in the CI

**Comment by MarkSymsCtx** [comment]
> This is all test code so I assume any failures would've shown up in the CI

Indeed, all test code is required to be 100% covered.

**Review by rosslagerwall — APPROVED** [review]

**Review by TimSmithCtx — APPROVED** [review]

**Pull request merged**